app/middleware/logZeroResults.js

```diff
@@ -2,10 +2,10 @@ const log = require('../lib/logger');
 
 function logZeroResults(req, res, next) {
   const search = res.locals.search;
-  const postcode = res.locals.postcode;
+  const postcodeSearch = res.locals.postcodeSearch;
 
   if (res.locals.gps.length === 0) {
-    log.warn({ search }, `No results were found for: { search: '${search}', postcode: '${postcode}'`);
+    log.warn({ search }, `No results were found for: { search: '${search}', postcode: '${postcodeSearch}'`);
```

Comment:
Agreed.

Also, I don't believe this middleware is required. Zero results is just a special case of the logging the number of results. I think the postcodeLookup middleware should log the number of results and the variables used in the query to arrive at that number of results.